utils/go-yq/wrapper.py

```
@@ -0,0 +1,77 @@
+
+subcommand = snakemake.params.get("subcommand", "")
+# yq/jq expression should be quoted
+expression = shlex.quote(snakemake.params.get("expression", ""))
+
+# Handle the case use creates a file from command line
+infile = snakemake.input
+if len(infile) == 0:
+    infile = " --null-input "
+
+# Handling file splitting
+outfile = snakemake.output
```

Ruff (0.14.7)

Undefined name `snakemake`

(F821)